osu.Game/Overlays/Settings/Sections/Input/MouseSettings.cs

```diff
@@ -14,9 +14,24 @@ public class MouseSettings : SettingsSubsection
     {
         protected override string Header => "Mouse";
 
+        private readonly BindableBool rawInputToggle = new BindableBool();
+        private Bindable<string> activeInputHandlers;
+
         [BackgroundDependencyLoader]
         private void load(OsuConfigManager osuConfig, FrameworkConfigManager config)
         {
+            activeInputHandlers = config.GetBindable<string>(FrameworkSetting.ActiveInputHandlers);
+            rawInputToggle.Value = activeInputHandlers.Value.Contains("Raw");
+            rawInputToggle.ValueChanged += enabled =>
+            {
+                const string raw_mouse_handler = @"OpenTKRawMouseHandler";
+                const string standard_mouse_handler = @"OpenTKMouseHandler";

```

Comment:
this is very temporary code. we can't reference by type because InputHandlers are `internal` and also in the desktop project.